### ISSUE TITLE
2.41.0 release broke global installation in CI environment

### ISSUE THREAD

**Issue by Overdrivr** [netlify/cli#798]
**- Do you want to request a _feature_ or report a _bug_?**
Bug

**- What is the current behavior?**
Crash during global installation

**- If the current behavior is a bug, please provide the steps to reproduce.**

We're using the netlify-cli in a Gitlab CI environment, yesterday build was passing with 2.40, today it's crashing with 2.41. I had to force version 2.40 to restore build.

The failure happens when installing netlify-cli globally. This is a simplified CI script to reproduce:

```
stages:
  - deploy

deploy_task:
  stage: deploy
  image: node:10
  script:
    - npm i -g netlify-cli
```

This is the error log:

```
$ npm i -g netlify-cli
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
/usr/local/bin/ntl -> /usr/local/lib/node_modules/netlify-cli/bin/run
/usr/local/bin/netlify -> /usr/local/lib/node_modules/netlify-cli/bin/run
> grpc@1.24.2 install /usr/local/lib/node_modules/netlify-cli/node_modules/grpc
> node-pre-gyp install --fallback-to-build --library=static_library
node-pre-gyp WARN Using needle for node-pre-gyp https download 
node-pre-gyp WARN Pre-built binaries not installable for grpc@1.24.2 and node@10.19.0 (node-v64 ABI, glibc) (falling back to source compile with node-gyp) 
node-pre-gyp WARN Hit error EACCES: permission denied, mkdir '/usr/local/lib/node_modules/netlify-cli/node_modules/grpc/src/node' 
gyp WARN EACCES current user ("nobody") does not have permission to access the dev dir "/root/.cache/node-gyp/10.19.0"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/local/lib/node_modules/netlify-cli/node_modules/grpc/.node-gyp"
gyp WARN install got an error, rolling back install
gyp WARN install got an error, rolling back install
gyp ERR! configure error 
gyp ERR! stack Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/netlify-cli/node_modules/grpc/.node-gyp'
gyp ERR! System Linux 4.19.78-coreos
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "--fallback-to-build" "--library=static_library" "--module=/usr/local/lib/node_modules/netlify-cli/node_modules/grpc/src/node/extension_binary/node-v64-linux-x64-glibc/grpc_node.node" "--module_name=grpc_node" "--module_path=/usr/local/lib/node_modules/netlify-cli/node_modules/grpc/src/node/extension_binary/node-v64-linux-x64-glibc" "--napi_version=5" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v64"
gyp ERR! cwd /usr/local/lib/node_modules/netlify-cli/node_modules/grpc
gyp ERR! node -v v10.19.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --library=static_library --module=/usr/local/lib/node_modules/netlify-cli/node_modules/grpc/src/node/extension_binary/node-v64-linux-x64-glibc/grpc_node.node --module_name=grpc_node --module_path=/usr/local/lib/node_modules/netlify-cli/node_modules/grpc/src/node/extension_binary/node-v64-linux-x64-glibc --napi_version=5 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v64' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/netlify-cli/node_modules/grpc/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:982:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
node-pre-gyp ERR! System Linux 4.19.78-coreos
node-pre-gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/netlify-cli/node_modules/grpc/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build" "--library=static_library"
node-pre-gyp ERR! cwd /usr/local/lib/node_modules/netlify-cli/node_modules/grpc
node-pre-gyp ERR! node -v v10.19.0
node-pre-gyp ERR! node-pre-gyp -v v0.14.0
node-pre-gyp ERR! not ok 
Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --library=static_library --module=/usr/local/lib/node_modules/netlify-cli/node_modules/grpc/src/node/extension_binary/node-v64-linux-x64-glibc/grpc_node.node --module_name=grpc_node --module_path=/usr/local/lib/node_modules/netlify-cli/node_modules/grpc/src/node/extension_binary/node-v64-linux-x64-glibc --napi_version=5 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v64' (1)
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/netlify-cli/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! grpc@1.24.2 install: `node-pre-gyp install --fallback-to-build --library=static_library`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the grpc@1.24.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-04-08T09_05_46_091Z-debug.log
```

Forcing 2.40 solves the issue as a workaround:

```
stages:
  - deploy

deploy_task:
  stage: deploy
  image: node:10
  script:
    - npm i -g netlify-cli@2.40.0
```

**- What is the expected behavior?**

I expect the package to get installed just fine.

**- Local Environment Information**

Everything is mentionned above

**Comment by bradlocking** [comment]
Also managing to replicate. running same install script from CI environment `npm i -g netlify-cli` and locking to `v2.40.0` is patching it for the time being

**Comment by mabhub** [comment]
Same here!
`node 12.16.1 / npm 6.13.4`

**Comment by drewm** [comment]
Same problem (and solution) reproduced. `node 13.6.0` and `npm 6.13.4`.

**Comment by simonfr** [comment]
Same problem : `node 13.11.0 / npm 6.13.7`

**Comment by RaeesBhatti** [comment]
We've reverted the changes related to logs tailing which require a native module, the cause of this problem.
https://github.com/netlify/cli/pull/799

**Comment by RaeesBhatti** [comment]
The reversion/fix has been released in netlify-cli@2.42.0. Please update and let us know if it is still an issue.

**Comment by marketerly** [comment]
Hi, I'm beginner, and I tried the following commands to install netlify-cli with no success yet:

npm i -g netlify-cli@2.40.0

npm i -g netlify-cli@2.42.0


This is what I got:

npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm ERR! Unexpected end of JSON input while parsing near '...GFMhqH7w9Etu5QHvvC7Cz'

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/taha/.npm/_logs/2020-06-18T10_34_36_562Z-debug.log

I'm following a tutorial, and get stuck at this step.

Note: I use  Node v12.16.3 and npm v6.14.4

Thanks,
